src/main/java/org/openrewrite/staticanalysis/csharp/CSharpFileChecker.java

```diff
@@ -19,15 +19,18 @@
 import org.openrewrite.Tree;
 import org.openrewrite.TreeVisitor;
 import org.openrewrite.csharp.tree.Cs;
+import org.openrewrite.internal.ReflectionUtils;
 import org.openrewrite.marker.SearchResult;
 
 /**
  * Add a search marker if vising a CSharp file
  */
 public class CSharpFileChecker<P> extends TreeVisitor<Tree, P> {
+    private static final boolean IS_CSHARP_AVAILABLE = ReflectionUtils.isClassAvailable("org.openrewrite.csharp.tree.Cs");
```

Comment:
Let's make an `isLanguageAvailable` function part of `ReflectionUtils` or otherwise available in the rewrite core modules as this functionality will end up used in many places

Reply:
@jkschneider did already partly suggest that, as I moved the `isClassAvailable` to the core:
- https://github.com/openrewrite/rewrite/pull/4810

Are you saying you want to go one step further, by creating a `isLanguageAvailable(Enum language)` kind of function in the core?